### DISCUSSION TITLE
Feature Request: Alpha Scrubber for Long Scrolling Assets and Channels/Structure Sections

### DISCUSSION THREAD

**Discussion by bgarrant** [craftcms/cms#8417] [Ideas]
When working on Craft sites with a lot of entries or assets, the scrolling can become an issue quickly.  For example, If I do not know the file name but I want to scroll down quickly to a few possible starting letters like 'V' it can be very slow with the "lazy-loading".  Pagination would be even worse.  By implementing a vertical Alpha Scrubber we could quickly slide with a mouse or touch on mobile to a specific letter.  I think this would be a great feature for Craft 3.  If Google Photos can do it with galleries in the tens of thousands we can surely do it with Craft.

Something like this with capital letters maybe.

![scrubber](https://cloud.githubusercontent.com/assets/2436728/24713796/a5425350-19f4-11e7-9d66-189254592dbc.PNG)

Please consider.

**Comment by bgarrant** [comment]
This may not be the perfect solution but we could definitely use a better way

**Comment by bgarrant** [comment]
This may be even better option: https://github.com/craftcms/cms/issues/818.  Same core issue though

**Comment by sjelfull** [comment]
Yeah, #818 generally needs to happen, but this sounds like a interesting addition. Maybe not like this, though.

Flexible filtering might help?

**Comment by ryanmasuga** [comment]
I agree, #818 needs to happen (if I do say so myself...) but I like seeing these other suggestions as well. I like the idea of flexible filtering - it's not even easy to filter by entry types at this point.

**Comment by brandonkelly** [comment]
Also related: #1630
